.github/BOTMETA.yml

```
@@ -0,0 +1,23 @@
+notifications: true
```

```suggestion
notifications: true
```
is this file needed? Maintainers could just subscribe to this repo. If it's needed, the bot should be configured for the repo

---

i guess the ignore.txt files can also be removed

---

> > @Andersson007 could you also add "rkpobe" to this repo?
> 
> Invited them with "maintainer" permissions
> 
> BTW @tbludau 's invitation is still pending. Is he still interested in maintaining this collection or we can remove him?
Hi @Andersson007,
i think tbludau is not interested any more. So yes you can remove him.

---

Thank you for the input i removed the files :-)

---

You're welcome! FYI I cancelled the invite